custom-elements/Document-createElementNS-customized-builtin.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
naming nit with the file, you named it `-customized-builtin` as opposed to `-customized-builtins` like the other PRs. Consistency would be nice.

Reply:
Fixed.